### PR TITLE
enhance experience with copyToClipboard

### PR DESCRIPTION
**Pull request by tearstains**
@ArvinH

### CODE REVIEW

File: devtools.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <html>
 <body>
-<script src="ruleMap.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.5/lodash.min.js"></script>
+<script>
```
Comment (ArvinH) [inline devtools.html]: This could be removed

File: sidebar.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<script src="ruleMap.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.5/lodash.min.js"></script>
+
+<div id="app" class="container mt-3">
+    hello world
```
Comment (ArvinH) [inline sidebar.html]: Let's put some fun stuff here 🚀

File: sidebar.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+(function () {
+
+    var page_generateAtomicClass = function (selectedElementCssText) {
```
Comment (ArvinH) [inline sidebar.js]: It would be appreciated if you could help converse all es5 to es6!

**Review by ArvinH — APPROVED** [review]
Thanks for the awesome contribution!!!!!!
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🏆 🏆 🏆 🏆 🏆 🏆

**Comment by tearstains** [comment]
card

**Pull request merged**